plugins/statistics.koplugin/main.lua

```diff
@@ -376,13 +401,17 @@ function ReaderStatistics:getDatesFromAll(sdays, ptype)
             table.sort(sorted_performance_in_pages)
             for i, n in pairs(sorted_performance_in_pages) do
                 if ptype == "daily_weekday" then
-                    date_text = os.date("%Y-%m-%d (%a)", n)
+                    date_text = string.format("%s (%s)",
+                        os.date("%Y-%m-%d ", n),
+                        shortDayOfWeekTranslation[os.date("%a", n)])
                 elseif ptype == "daily" then
                     date_text = os.date("%Y-%m-%d" , n)
                 elseif ptype == "weekly" then
-                    date_text = os.date("%Y Week %W" , n)
+                    date_text = os.date("%Y ".. week_translate .. " %W" , n)
                 elseif ptype == "monthly" then
-                    date_text = os.date("%B %Y" , n)
+                    month_translate = monthTranslation[os.date("%B" , n)]
+                    date_text = os.date(" %Y" , n)
+                    date_text = month_translate .. date_text
```

Comment:
It's not your fault since you are just following the style from the existing code. But the rest of the code base only puts a space after the `,` in function calls, not before. Would be nice if you can also update it since you are at it :)